pkgs/development/python-modules/python-jose/default.nix

```
@@ -23,6 +23,11 @@ buildPythonPackage rec {
     cryptography # optional dependency, but needed in tests
   ];
 
+  patches = [
+    # relax ecdsa deps
+    ./ecdsa.patch
+  ];
```

```suggestion
  patchPhase = ''
    substituteInPlace requirements.txt \
      --replace 'ecdsa<0.15' 'ecdsa' \
      --replace 'ecdsa <0.15' 'ecdsa'
  '';
```

and the patch can be removed. This breaks less often and is better for simple changes like this.

---

thanks :D didnt know this was possible